build.gradle

```
@@ -98,15 +98,16 @@ tasks.withType(Sign).configureEach {
     notCompatibleWithConfigurationCache("$name task does not support configuration caching")
 }
 
-def gitHubReleaseName() {
-    return version.toString()
-}
-
 def gitReleaseTag() {
-    return "v${version}"
+    return version.map { it -> "v$it" }
```

Does Groovy have an implicit `it` like Kotlin? If so, you could simplify this to:

```suggestion
    return version.map { "v$it" }
```

---

Ah yup it does